### ISSUE TITLE
Windows build fails because of missing curl debug libs

### ISSUE THREAD

**Issue by robbiesri** [pierricgimmig/orbitprofiler#143]
When running bootstrap-orbit.bat, I hit this:

```
CMake Error at external/vcpkg/installed/x86-windows/share/curl/libcurl-target.cmake:78 (message):
  The imported target "CURL::libcurl" references the file

     "C:/src/git/orbitprofiler/external/vcpkg/installed/x86-windows/debug/lib/libcurl-d_imp.lib"

  but this file does not exist.  Possible reasons include:

  * The file was deleted, renamed, or moved to another location.

  * An install or uninstall procedure did not complete successfully.

  * The installation package was faulty and contained

     "C:/src/git/orbitprofiler/external/vcpkg/installed/x86-windows/share/curl/libcurl-target.cmake"

  but not all the files it references.

Call Stack (most recent call first):
  external/vcpkg/installed/x86-windows/share/curl/curl-config.cmake:20 (include)
  external/vcpkg/scripts/buildsystems/vcpkg.cmake:274 (_find_package)
  CMakeLists.txt:27 (find_package)


CMake Error at external/vcpkg/scripts/buildsystems/vcpkg.cmake:274 (_find_package):
  Found package configuration file:

    C:/src/git/orbitprofiler/external/vcpkg/installed/x86-windows/share/curl/curl-config.cmake

  but it set CURL_FOUND to FALSE so package "CURL" is considered to be NOT
  FOUND.
Call Stack (most recent call first):
  CMakeLists.txt:27 (find_package)


-- Configuring incomplete, errors occurred!
```

**Comment by robbiesri** [comment]
After snooping around, it looks like the expected name for the curl lib had caused issues for other people, and it was fixed here: https://github.com/microsoft/vcpkg/pull/5968/files

From looking at this, I saw that the version this was fixed in was 7.61.1-4. When I looked at the version that vcpkg installed, I got this:
```
PS C:\src\git\orbitprofiler\external\vcpkg> .\vcpkg.exe list curl
curl:x64-windows                                   7.61.1-2         A library for transferring data with URLs
curl:x86-windows                                   7.61.1-2         A library for transferring data with URLs
```

So I'll try to upgrade my local curl, and see what happens. But then this upgrade (if needed) should be run by some portion of the build chain. I'm kinda surprised vcpkg wouldn't handle this.
